packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -206,6 +206,9 @@
             "type": "boolean",
             "nullable": false,
             "description": "Can only be set to false for an insecure grpc connection."
+          },
+          "monitorMode": {
```

Comment:
@StepniowskiP `mode` implies to me an enum not a boolean, could you add a description to the spec here that explains what this does and what are the possible values (`true` and `false`) change in the behavior?
If it is a boolean then maybe call it `monitorModeEnabled` and if it has more possible values other than true/false then make it an enum so that later on those other possible values can be added as needed without having to break backwards compatibility.

Reply:
I have added description to this parameter